hmosutils/src/main/java/me/jfenn/hmosutils/ColorUtils.java

```diff
@@ -0,0 +1,103 @@
+package me.jfenn.hmosutils;
+
+import ohos.agp.colors.HsvColor;
+import ohos.agp.colors.RgbColor;
+import ohos.agp.utils.Color;
+
+
+/**
+ * ColorUtils.
+ */
+
```

Comment:
Unnecessary whitespaces/lines must be removed.

Reply:
Done